packages/mongodb-chatbot-eval/testData/config/configTest.cjs

```diff
@@ -4,4 +4,7 @@ module.exports = async () => ({
   evaluationStore: {},
   reportStore: {},
   commands: [],
+  async afterAll() {
+    "foo";
```

Comment:
```suggestion
    console.log("afterAll");
```

Kind of a weird spot for a (non-functional) directive prologue? Maybe we can just do a console log instead.

Reply:
lets not do a console.log b/c it'd show up in the test logs. 

i put the `"foo"` b/c the func needs a body to be valid. i can comment something to that affect.

Reply:
Oh makes sense. I think you could also just use a literal comment instead of the string. Doesn't really matter though.